.github/ISSUE_TEMPLATE/usage_question.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Usage question
+about: If you have a usage question
+title: "[SO]"
+labels: question
+assignees: ''
+
+---
+
+** If your issue is a usage question, submit it here instead:**
+- **StackOverflow with the scikit-learn tag: https://stackoverflow.com/questions/tagged/scikit-learn**
+- **Gitter: https://gitter.im/scikit-learn/scikit-learn**
+- **Mailing List: https://mail.python.org/mailman/listinfo/scikit-learn**
```

Comment:
Just a comment / question : the order means a priority? Which of those channels is the most reliable for the core-devs? That one should go first, probably...

Reply:
I did not put any order indeed.

Even if I am sometimes disappointed with some SO answers, this might be the channel the most reliable to get an answer fast (maybe the wrong one :)).

Reply:
I put it in the order that I would think the best but this is opened to discucssion :)

Reply:
I think we should favor SO first because of google-ability and reuse of past answers.